ipywidgets/widgets/interaction.py

```diff
@@ -52,8 +52,8 @@ def _get_min_max_value(min, max, value=None, step=None):
         if not isinstance(value, Real):
             raise TypeError('expected a real number, got: %r' % value)
         if not value:
```

Comment:
Can you make this `if value == 0`? It makes it a little clearer to me that this is definitely a number and not the None condition above.